ekm/signer_key_manager_test.go

```diff
@@ -788,3 +790,133 @@ func TestEkmListAccounts(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, len(accounts))
 }
+
+func TestConcurrentSlashingProtectionAttData(t *testing.T) {
+	require.NoError(t, bls.Init(bls.BLS12_381))
+
+	km := testKeyManager(t, nil)
+
+	sk1 := &bls.SecretKey{}
+	require.NoError(t, sk1.SetHexString(sk1Str))
+	require.NoError(t, km.AddShare(sk1))
```

Comment:
we should add another key here, or even make a slice of keys and run X goroutines for each

because EKM locks per key, then this would test multiple locks which greatly increases the coverage